src/JsonDeserializer.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\AzureStorageTableExtractor;
+
+use GuzzleHttp\Psr7\Stream;
+use Keboola\Component\JsonHelper;
+use MicrosoftAzure\Storage\Table\Internal\TableResources as Resources;
+use MicrosoftAzure\Storage\Table\Models\Entity;
+use RuntimeException;
+use MicrosoftAzure\Storage\Table\Internal\IODataReaderWriter;
+
+/**
+ * Our version of the MicrosoftAzure\Storage\Table\Internal\JsonODataReaderWriter
+ * We do not need to map the results to entities.
+ */
+class JsonDeserializer implements IODataReaderWriter
```

Comment:
Microsoft API by default mapuje response z API na entity objekty, ... nam uplne staci decodovany JSON, je to tak rychlejsie.